### DISCUSSION TITLE
LEDs

### DISCUSSION THREAD

**Discussion by frank-tanne** [hoylabs/OpenDTU-OnBattery#1195] [General]
Was haben eigentlich die zwei LED für funktion die man anschließen kann.

**Comment by schlimmchen** [comment]
https://www.opendtu.solar/hardware/led/

=> Ich kanns es dir auch nicht sagen. Auf dem OpenDTU Fusion board sieht man es: Die eine zeigt eine Verbindung zum MQTT Broker an, jedenfalls laut Beschriftung der LED. Die andere ob ein (alle?) inverter online sind, also strom produzieren? also wenn einer erreichbar ist und strom produziert reicht das jedenfalls nicht, dass die LED leuchtet...
